### DISCUSSION TITLE
Ability to re-request review in the GitHub Mobile app

### DISCUSSION THREAD

**Discussion by cristeahub** [community/community#123938] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Hello!

GitHub mobile has quickly become an essential tool in my daily workflow.

Recently, my team and I have leaned more into using the github way of developing more and more. This includes reviewing and re-requesting review when updates come.

The issue I face is that when I am on GitHub mobile I am able to dismiss a review, but not able to quickly re-request a review.

I currently unassign and assign a reviewer to simulate a re-request, however being able to re-request directly would be vastly easier and more streamlined.

**Comment by alcere** [comment]
Hi @cristeahub! Thank you for your feedback! We are glad to be helpful in your daily work. Can you confirm that you are an iOS user, please? Thanks!

**Reply by cristeahub** [reply]
Yes, I am an iOS user and am using the latest TestFlight build as of writing this (1.163.0 131249690)

**Reply by alcere** [reply, marked as answer]
Thanks for the details! I will open a feature request and let the team prioritize this work!

**Comment by tonka3000** [comment]
Same issue on the android version as well. Re-request is not available next to the reviewer.

**Reply by alcere** [reply]
Hi @tonka3000! This looks like a bug. Thanks for reporting it! Let me please double check the steps you are following so I can better track this problem.

Normally, within a pull request, after expanding the `Reviewers` section, you should see the list of reviewers. Each reviewer has an overflow button that presents a pop-up menu with the option to 'View review' and `Re-request` if they already left a review. Don't you see this overflow button, or only the `Re-request` option?

**Reply by tonka3000** [reply]
I can't see any re-request button. On the website it is shown.

![Screenshot_20240916_184456_GitHub.jpg](https://github.com/user-attachments/assets/94c6e097-db6e-47ca-b22d-c76aa195610f)

**Reply by alcere** [reply]
Thanks for sharing @tonka3000, and we are sorry for the troubles. I will create a bug and we will try to get it fixed as soon as possible.

**Reply by alcere** [reply]
Just one more question to narrow down the problem, can you re-request reviews for that same pull request using GitHub.com web page?

**Reply by cristeahub** [reply]
This is also the case on iOS to be fair:

From the app:
<img width="300" alt="image" src="https://github.com/user-attachments/assets/e20b6634-cc50-4522-9bcd-4f631a69a986">

On web, here I can even re-request review from the commented review:
<img width="418" alt="image" src="https://github.com/user-attachments/assets/019031d9-5476-47e0-9d99-a31755b3f8a6">

I'm still on iOS
<img width="225" alt="image" src="https://github.com/user-attachments/assets/a6fd9944-a2d3-4584-adbc-b3e67d21c9db">

**Reply by alcere** [reply]
Thanks for the info @cristeahub!

**Reply by tonka3000** [reply]
> Thanks for sharing @tonka3000, and we are sorry for the troubles. I will create a bug and we will try to get it fixed as soon as possible.

@alcere Yes, I can do it on github.com

**Comment by joeeekaaa-cmd** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Hello!
> 
> GitHub mobile has quickly become an essential tool in my daily workflow.
> 
> Recently, my team and I have leaned more into using the github way of developing more and more. This includes reviewing and re-requesting review when updates come.
> 
> The issue I face is that when I am on GitHub mobile I am able to dismiss a review, but not able to quickly re-request a review.
> 
> I currently unassign and assign a reviewer to simulate a re-request, however being able to re-request directly would be vastly easier and more streamlined.

**Comment by joeeekaaa-cmd** [comment]
https://github.com/orgs/community/discussions/123938#discussion-6672328
